README.md

```diff
@@ -11,13 +11,14 @@ $ npm install uniflow --save
 ## Usage
 
 ```js
-var Uniflow = require('uniflow')
+var createActions = require('uniflow/createActions')
+var createStore = require('uniflow/createStore')
```

Comment:
This is a very uncommon pattern. The primary use of a library should not require knowing sub-modules within the library. I prefer what was there before.